types/index.d.ts

```diff
@@ -0,0 +1,11 @@
+// TypeScript Version: 3.5
+
+import {Plugin} from 'unified'
+import {HastUtilToHtmlOptions} from 'hast-util-to-html'
+
+interface htmlOptions extends HastUtilToHtmlOptions {
+  sanitize?: boolean | {[key: string]: any}
```

Comment:
This is a good starting point, as mentioned in the description, it is a bit broad.
The preference in Unified/Remark is to get upstream typings in place, then leverage them where it makes sense.

>  If needed, I can try and write a .d.ts file for that repo as well, however, I would like to try and solve one step at a time, and make sure that I am doing things correctly along the way

Thanks! This would be appreciated

Reply:
No problem! I will go ahead and get started!

Reply:
Also #33 reminded me there is a `handlers` property, which is based off https://github.com/syntax-tree/mdast-util-to-hast/blob/81ae2dde0ac255d413df0ca7bf20e2ceae1b870e/types/index.d.ts#L12-L22 which we may want to add

Reply:
Good catch! I had to add type definitions for [hastscript](https://github.com/syntax-tree/hastscript) before I can do the type definitions for hast-util-sanitize, so once those two PRs go through I'll update this one to include both `sanitize` and `handlers`